xml/operations-compute-gpu_passthrough.xml

```diff
@@ -0,0 +1,272 @@
+            QEMU: Checking if IOMMU is enabled by kernel
+            : WARN (IOMMU appears to be disabled in kernel. Add intel_iommu=on to kernel cmdline arguments)
+            .....
+          </screen>
+          <para>
+            To modify the kernel cmdline as suggested in the warning, edit the file
+            <filename>/etc/default/grub</filename>
+            and append
+            <literal>intel_iommu=on</literal>
+            to the
+            <literal>GRUB_CMDLINE_LINUX_DEFAULT</literal>
+            variable. Then run
+            <literal>update-bootloader</literal>.
+          </para>
+          <para>
+            A reboot will be required for <literal>iommu_groups</literal> to be
+            enabled.
+          </para>
+        </step>
+        <step>
+          <para>
+            After the reboot, check that IOMMU is enabled:
+          </para>
+          <screen>&prompt.root;virt-host-validate
+            .....
+            QEMU: Checking if IOMMU is enabled by kernel
+            : PASS
+            .....
+          </screen>
+        </step>
+        <step>
+          <para>
+            Confirm IOMMU groups are available by finding the group associated with
+            your PCI device (for example Nvidia GPU):
+          </para>
+          <screen>&prompt.ardana;lspci -nn | grep -i nvidia
+            84:00.0 3D controller [0302]: NVIDIA Corporation GV100GL [Tesla V100 PCIe 16GB] [10de:1db4] (rev
+            a1)
+          </screen>
+          <para>
+            In this example, <literal>84:00.0</literal> is the address of the PCI device. The vendorID
+            is <literal>10de</literal>. The productID is <literal>1db4</literal>.
+          </para>
+        </step>
+        <step>
+          <para>
+            Confirm that the devices are available for passthrough:
+          </para>
+          <screen>&prompt.ardana;ls -ld /sys/kernel/iommu_groups/*/devices/*84:00.?/
+            drwxr-xr-x 3 root root 0 Nov 19 17:00 /sys/kernel/iommu_groups/56/devices/0000:84:00.0/
+          </screen>
+        </step>
+      </procedure>
+    </listitem>
+    <listitem>
+      <para>
+        <emphasis role="bold">Preparing &o_comp; via the input model updates</emphasis>
+      </para>
+      <para>
+        To implement the configuration you require, log into the &clm; node and update
+        the &clm; model files to enable GPU passthrough for compute nodes.
+      </para>
+      <para>
+        <emphasis role="bold">Edit servers.yml</emphasis>
+      </para>
+      <para>
+        Add the pass-through section after the definition of servers section in the yaml file.
+        Here's an example:
+      </para>
+      <screen>
+
+        ---
+        product:
+        version: 2
+
+        baremetal:
+        netmask: 255.255.255.0
+        subnet: 192.168.100.0
+
+
+        servers:
+
+        .
+        .
+        .
+        .
+
+        pass-through:
+          servers:
+            - id: compute-0001
```

Comment:
We need to emphasise that node id names here should match the corresponding node id names in the servers: table above... This may already be addressed below... If not then perhaps we should have a partial entry in the example servers: table above for each of the nodes we list in this pass-through servers table?

Reply:
Added additional sections in the example servers.yml